dotnet/src/Azure.Iot.Operations.Services/Azure.Iot.Operations.Services.csproj

```diff
@@ -27,4 +27,18 @@
     <DelaySign>false</DelaySign>
   </PropertyGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Azure.Iot.Operations.Services.UnitTests" />
+    <InternalsVisibleTo Include="Azure.Iot.Operations.Connector.UnitTests" />
+  </ItemGroup>
```

Comment:
This may be a relic of a merge issue. These internalsVisibleTo declarations live elsewhere now